demos/general/demo_OnACID_mesoscope.py

```diff
@@ -34,16 +36,18 @@
 from caiman.source_extraction import cnmf as cnmf
 from caiman.utils.utils import download_demo
 
+%matplotlib qt 
```

Comment:
We can't do this; it is invalid syntax outside of ipython

Reply:
Actually, we may be able to do this with eval(), conditionally on if we're in ipython

Reply:
Good idea I'll give it a shot.